scripts/environment.sh

```
@@ -0,0 +1,22 @@
+    elif curl -s http://localhost:3001/health > /dev/null 2>&1; then
+        CURRENT_ENV=$(curl -s http://localhost:3001/health | jq -r '.environment')
+    else
+        CURRENT_ENV="unknown"
+    fi
+
+    if [ "$CURRENT_ENV" == "green" ]; then
+        TARGET_ENV="blue"
+        TARGET_PORT=3001
+    else
+        TARGET_ENV="green"
+        TARGET_PORT=3000
```

Shellcheck (0.10.0)

[warning] TARGET_PORT appears unused. Verify use (or export if used externally).

(SC2034)